spec/factories/users.rb

```diff
@@ -55,6 +55,21 @@
     [Faker::Name.first_name, Faker::Name.last_name, Faker::Internet.domain_word, n].join("-")
   end
 
+  factory :admin_v2, class: User do
```

Comment:
Had to create a new admin factory because the existing one bakes in the notion of an org that the user has user_permissions of.